mix.exs

```diff
@@ -47,11 +47,11 @@ defmodule PromEx.MixProject do
       # Required dependencies
       {:jason, "~> 1.2"},
       {:finch, "~> 0.8.0"},
-      {:telemetry, ">= 0.4.0"},
-      {:telemetry_poller, "~> 0.5.1"},
-      {:telemetry_metrics, "~> 0.6.0"},
-      {:telemetry_metrics_prometheus_core, "~> 1.0.1"},
-      {:plug_cowboy, ">= 2.4.0"},
+      {:telemetry, "~> 1.0.0"},
+      {:telemetry_poller, "~> 1.0.0"},
+      {:telemetry_metrics, "~> 0.6.1"},
+      {:telemetry_metrics_prometheus_core, "~> 1.0.2"},
+      {:plug_cowboy, "~> 2.5.1"},
```

Comment:
just a FYI: this makes it incompatible with Phoenix 1.5 apps because 
https://github.com/phoenixframework/phoenix/blob/v1.5/mix.exs#L73